src/ucp/api/ucp.h

```diff
@@ -2100,6 +2137,35 @@ ucs_status_t ucp_context_query(ucp_context_h context_p,
                                ucp_context_attr_t *attr);
 
 
+/**
+ * @ingroup UCP_MEM
+ * @brief Compare two remote keys according to parameters
+ *
+ * This routine returns equal if all the transports can compare equal and
+ * configurations are equal. At least one compare mode is needed.
+ *
+ * The comparison result is < 0 if rkey1 is lower than rkey2, 0 if they
+ * are equal or > 0 if rkey1 is greater than rkey2.
+ *
+ * If rkey2 contains one or more transports that are not comparable,
```

Comment:
what does it mean "transport that is not comparable"?

Reply:
removed as per [comment](https://github.com/openucx/ucx/pull/9289#discussion_r1297110413)